### DISCUSSION TITLE
Watching a specific branch

### DISCUSSION THREAD

**Discussion by kaikiat** [argoproj/argo-cd#10268] [General]
With reference to this file : https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/application.yaml

Is it possible for argo-cd to watch a specific branch ? If so which parameter do we change in this manifest file? Thank you!

**Comment by crenshaw-dev** [comment]
It is! Set the `targetRevision` field to the branch name. https://github.com/argoproj/argo-cd/blob/4dc276e269d5d2eab981caa1bcad0d713cecaebd/docs/operator-manual/application.yaml#L20
